elyra/pipeline/kfp/kfp_component_utils.py

```diff
@@ -19,7 +19,7 @@
             "type": "string"
         },
         "description": {
-            "type": "string"
+            "type": ["string", "null"]
```

Comment:
why we use "null" instead of `null`?

Reply:
Strangely, that's how JSON schemas handles the 'null' case: https://json-schema.org/understanding-json-schema/reference/null.html

Reply:
Thanks for the clarification, I was researching only the json part of it and not the jsonSchema.